boa_profiler/src/lib.rs

```diff
@@ -24,7 +24,7 @@
 //! [boa-web]: https://boa-dev.github.io/
 //! [boa-playground]: https://boa-dev.github.io/boa/playground/
 
-#![cfg_attr(not(test), forbid(clippy::unwrap_used))]
+#![cfg_attr(not(any(test, feature = "profiler")), forbid(clippy::unwrap_used))]
```

Comment:
Let's replace the `unwrap`s with `expects`s, then this change can also be reverted.